client/src/components/Example.js

```diff
@@ -1,87 +1,31 @@
 import React, { Component } from "react";
-import SimpleStorageContract from "../contracts/SimpleStorage.json";
-import getWeb3 from "../getWeb3";
 /*
 import Web3 from "web3";
 import Web3Modal from "web3modal";
 */
+import { Button, Jumbotron } from "react-bootstrap";
+import { Link } from "react-router-dom";
 import "./Example.css";
 
 class Example extends Component {
```

Comment:
Do we want to rename "Example" to "Home"? Applies to the file name, component class, and css file. I initially called it example to house the auto-generated sample code.

Reply:
Good idea.